### ISSUE TITLE
Mastodon theme has low contrast for the focus ring

### ISSUE THREAD

**Issue by nolanlawson** [nolanlawson/pinafore#1790]
This black seems almost indistinguishable from the background:

![Screenshot from 2020-05-23 15-25-39](https://user-images.githubusercontent.com/283842/82741690-bed33280-9d09-11ea-93ca-337311baa718.png)
